projects/sandbox/src/app/dataset-info/dataset-info.component.ts

```diff
@@ -99,4 +105,14 @@ export class DatasetInfoComponent extends SubscriptionManager {
   showProcessingErrors(): void {
     this.subs.push(this.modalConfirms.open(this.modalIdProcessingErrors).subscribe());
   }
+
+  /**
+   * convertUTCDateToLocalDate
+   * @param {string} dateIn
+   **/
+  convertUTCDateToLocalDate(dateIn: string) {
+    const date = new Date(dateIn);
+    const newDate = new Date(date.getTime() - date.getTimezoneOffset() * 60 * 1000);
```

Comment:
What are those magic numbers? 60 and 1000?
Could you create some class variables to represent what those numbers are?

Reply:
Local pre-computed variable added.